### ISSUE TITLE
Sketch Measure toolbars doesn't show in Sketch 45

### ISSUE THREAD

**Issue by Miiost** [utom/sketch-measure#330]
Hi!
Nothing happens when I try to open the toolbars (both). Dito when I try with ctrl + shift +B.
I tried to uninstall, re-install, quit, switch off the internet connection and whatever. I can't seem to be able to trigger any action from the plugin. Please help!

**Comment by KingWu** [comment]
Face the same issue

**Comment by brandbrilliance** [comment]
Same here. Sketch 45.1 breaks the plugin completely. Now I can't export measures anymore.

**Comment by kellierife** [comment]
Same. I cannot mark properties in Sketch 45.1. It does nothing when I run that action.

**Comment by brandbrilliance** [comment]
Also on older versions some stuff works but not all properties are possible in NOTES

**Comment by utom** [comment]
@kellierife @brandbrilliance That is fixed at some time ago~ get the new version
@Miiost Tell me your macOS version and Sketch version?

**Comment by brandbrilliance** [comment]
I'm using sketch 45.1 and the latest version 2.6.5. Unless you added a version today.

**Comment by brandbrilliance** [comment]
And I'm on Sierra latest version.

**Comment by utom** [comment]
Try to download from web, and install it

**Comment by utom** [comment]
Yep, I see that.  Any updates isn't working via the Sketch Updater. Try to download new version from web.

**Comment by dpelletier2017** [comment]
Same thing happens to me : 
- Sketch Measure 2.6.12
- Sketch 45.2
- macOS High Sierra Beta 4

**Comment by utom** [comment]
@dpelletier2017 Try to download new version from web.

**Comment by dpelletier2017** [comment]
Hi utom, thanks for the quick support :)

Unfortunately, I've downloaded it from the measure's website, from the GitHub link in the readme and from GitHub clone.
I've tried in Sketch 46 Beta because they seem to have repaired a lot of stuff related to plug-ins, but it hasn't worked either :-/ https://www.sketchapp.com/beta/

The plugin looks absolutely dope though, it's the best Sketch plugin and one of the finest works I've seen on the web.

**Comment by utom** [comment]
@dpelletier2017 I'm so sorry, I don't have macOS High Sierra Beta 4 to test the problem.
But you can try other version in here https://github.com/utom/sketch-measure/releases

**Comment by dpelletier2017** [comment]
@utom : 
Sketch 46 was just released and it works :)
It also worked on my MBP with OS X 10.11 on El Cap

**Comment by mayaelsibai** [comment]
Had this issue with sketch 49 and now that I have updated Sketch to version 50 it still does not work.
any other recommended apps?

**Comment by ewelinapio** [comment]
same here

**Comment by panchovm** [comment]
I have Sketch 50 and doesn't work anymore 😢. It is a great plugin that I use always.

**Comment by luxlogica** [comment]
@utom It's likely that this issue can be safely closed now.
